README.md

```diff
@@ -9,24 +10,67 @@ relation.
 
 ```javascript
 function mystery(n) {
-    if(n <= 1)
+    // Base case: if n is less than or equal to 1, return
+    if (n <= 1)
         return;
     else {
+        // Recursive call to function with n divided by 3
+        
         mystery(n / 3);
+
+        // Initialize count variable
         var count = 0;
+
+        // Another recursive call to mystery function with n divided by 3
+        
         mystery(n / 3);
-        for(var i = 0; i < n*n; i++) {
-            for(var j = 0; j < n; j++) {
-                for(var k = 0; k < n*n; k++) {
+
+        // Nested loop structure
+        // The outer loop runs n^2 times
+        for (var i = 0; i < n * n; i++) {
+            // The middle loop runs n times
+            for (var j = 0; j < n; j++) {
+                // The inner loop runs n^2 times
+                
+                for (var k = 0; k < n * n; k++) {
+                    
                     count = count + 1;
                 }
             }
         }
+
+        
         mystery(n / 3);
     }
 }
 ```
+The function makes three recursive calls to itself with the argument being $n/3$, $3T(n/3)$ the front "3" that is multiplying the $T(n/3)$ is there to account for the three recursive calls. The function also does a triple nested loop.
+So, the outer and iner loops each run $n^2$ times and then the middle loop runs $n$ times.
+that gives you $n^2 * n * n^2 = n^5$. Together you get the relation $T(n)=3T(n/3) + n^5$. This is my recurrance realtion. Because the nested loops dominate the time complexity the tight bound for Big $O$ is $\theta (n^5)$
+
+I used the recurrsion tree method to solve the reccurance relation:
+```
+
+The function makes 3 recursive calls to itself with the argument being n/3
+
+0: Level 0 = n^5 // This is the total work on each level
+1: Level 1 = 3*(n/3)^5 // Each level is multiplied by 3 becsaue of the three recursive calls
```

Comment:
Each level of recursion adds another term to a sum.

Reply:
So, the nested loops are $n^5$ and the functions 3 recursive calls are $n/3$.
Level 0: = $n^5$ from the nested loop
Level 1: = there are 3 calls and each call is equal to $n/3$ so you multiply $3*(n/3)^5$ = $n^5 / 3^5$
Level 2: Each of the previous 3 calls at level 1 do 3 more calls each equal to $(n/9)^5$ so the work done is $9 * (n/9)^5 = n^5 / 3^10$
This simplifies to $n^5 / 3^10$ because $9 = 3^2$ and $(9)^5 = 3^10$ so $(n/9)^5 = n^5 / 3^10$
Level 3: Each call is $(n/27)^5$ so you get $27 * (n/27)^5$ 
$27 * (n/27)^5 = 27 * (n^5 / 27^5) = n^5 / 27^4$
Then, $27 = 3^3$ and $27^4 = (3^3)^4 = 3^12$
So, $n^5 / 27^4 = n^5 / 3^12$

This is actually showing that the amount of work done at each level is decreasing and why the time complexity is equal to $O(n^5)$

Reply:
You're omitting a term from the previous expansion each time. The $n^5$ that is there initially stays there, as it's not part of the $T(...)$ expression. This happens for each expansion.

Reply:
So it would be
Level 0 = $n^5$
Level 1 = $n^5 / 3^5 +n^5$
Level 2 = $n^5 / 3^10 + n^5$
Level 3 = $n^5 / 3^15 + n^5$

Reply:
No, each level has an *additional* term from the expansion.